templates/view-submitted-response.html

```diff
@@ -29,8 +29,19 @@ <h1 class="u-mt-m">
   </h1>
   {{ onsMetadata(content.metadata)}}
   {% if not content.view_submitted_response.expired %}
+    {{
+      onsButton({
+        "type": 'button',
+        "text": 'Print answers',
+        "buttonStyle": "print",
+        "classes": "btn--small",
+        "attributes": {
+          "data-qa": "btn-print"
+        }
+      })
+    }}
     {% block summary %}
-      <div class="u-mb-m">
+      <div class="u-mt-l">
```

Comment:
You still want to keep the margin bottom, also this should really be added via the ons summary component, but we need to do some refactoring and clean up as section summary doesn't set these. For now, I think it is fine to add here.